src/containers/forms/ConnectedPlanForm/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
why not use `store.dispatch` in the `onSubmitSuccess` callback as an alternative to having to connect the form.

Reply:
@p-netm Ok....What are the pros of doing it like that over connect?

Reply:
@p-netm https://stackoverflow.com/a/41107592/3247914

Reply:
@kelvin-muchiri with respect to https://stackoverflow.com/a/41107592/3247914 if PlanForm is called from an already redux-connected component (which is the case 100% of the time) then it already has access to a dispatch function provided via `mapDispatchToProps` of the component that calls PlanForm, no?  If so I am uncertain whether we need `ConnectedPlanForm`